testing-scripts/bigtable/test_bigtable.py

```diff
@@ -0,0 +1,104 @@
+"""
+This module provides testing functionality of the BigTable Init Action.
+
+Test logic:
+1. Create test table and fill it with some data by injecting commands into hbase shell.
+2. Validate from local station that BigTable has test table created with right number of rows.
+
+Note:
+    Test REQUIRES cbt tool installed which provides CLI access to BigTable instances.
+    See: https://cloud.google.com/bigtable/docs/cbt-overview
+"""
+import unittest
+import random
+
+from parameterized import parameterized
+
+from dataproc_test_case import DataprocTestCase
+
+
+class BigTableTestCase(DataprocTestCase):
+    COMPONENT = 'bigtable'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/bigtable/bigtable.sh'
```

Comment:
All of these tests will eventually need to run against PRs (code that hasn't been checked in and synced to the GCS bucket). I think it makes sense for the tests to copy the scripts from their checkout (`../bigtable/bigtable.sh`) into a staging GCS bucket and use that

Reply:
They do. DataprocTestCase overwrites INIT_ACTION if the 'bucket' environment variable is set https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/355/files#diff-a2f3ab15b1953c7bdb4d44372493b727R47

We create a bucket and upload the pending tests in run_tests.sh
https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/355/files#diff-dadc96d79206c24169cd546cd80eece2R14

Reply:
Can we remove this INIT_ACTION variable just to make it clear then?